toolchain/driver/driver_test.cpp

```diff
@@ -228,5 +228,10 @@ TEST_F(DriverTest, FileOutput) {
   EXPECT_THAT(ReadFile("test.s"), ContainsRegex("Main:"));
 }
 
+TEST_F(DriverTest, LanguageServerNoStdin) {
+  EXPECT_FALSE(driver_.RunCommand({"language-server"}).success);
+  EXPECT_THAT(test_error_stream_.TakeStr(), HasSubstr("requires input_stream"));
+}
+
```

Comment:
Thanks! FYI, #4846 was also fixing this but that's still awaiting review. I'll probably replace this test after #4841 is in, per the TODO in #4846.